homeassistant/components/tado/config_flow.py

```diff
@@ -105,13 +105,6 @@ async def async_step_homekit(
         self._abort_if_unique_id_configured()
         return await self.async_step_user()
 
-    def _username_already_configured(self, user_input):
```

Comment:
Is this function never used?

Reply:
As far as I can judge and looking into a few other components, it didn't showed any active functionality. If I missed a part, please eloborate, whereas it's the first week I am actively getting into HA development. :)

Reply:
Yea it was just a question, I am on my phone so I couldn't see the whole file right now

Reply:
The CI isn't complaining about this function so I guess it is unused. But the CI is complaining about other stuff (like that missing Any) could you take a look?

Reply:
Yes, according to the reference check I have been using and snooping around in other other components I didn't see a smiliar kind of function. Also, when doing some functional testing this wasn't giving issues with a current installation and a brand new one, I just made.

Reply:
> The CI isn't complaining about this function so I guess it is unused. But the CI is complaining about other stuff (like that missing Any) could you take a look?

I'll check.

Reply:
> The CI isn't complaining about this function so I guess it is unused. But the CI is complaining about other stuff (like that missing Any) could you take a look?

Fixed. It missed the import. Now only the Black linter is being stubborn. The pre-commit run locally passed.

Reply:
`black .` should do the trick I think